src/views/inspector/InspectorViewProvider.ts

```diff
@@ -384,6 +384,7 @@ export class InspectorViewProvider implements vscode.WebviewViewProvider {
   }
 
   private async getFeatureEnvironmentStatusesHTML(folder: vscode.WorkspaceFolder) {
+    if (this.selectedType !== 'Feature') return ''
```

Comment:
should this be checking for 'Variable'?

Reply:
we don't want this to be evaluated for anything other than features, so I think this makes sense for now? ie. if we added "Environment" as a type this would still be true

Reply:
nvm just saw that it's `!==`